proxy/coordinator.go

```diff
@@ -13,13 +13,29 @@ import (
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/robustperception/pushprox/util"
 )
 
 var (
 	registrationTimeout = kingpin.Flag("registration.timeout", "After how long a registration expires.").Default("5m").Duration()
 )
 
+// Coordinator metrics.
+var (
+	knownClients = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name: "known_clients",
```

Comment:
Just "clients" would do I think. You can also use promauto to save the manual register.

Reply:
Fixed.